package.json

```diff
@@ -51,6 +51,7 @@
     "@types/lodash": "^4.17.12",
     "babel-jest": "^29.7.0",
     "grpc_tools_node_protoc_ts": "^5.3.3",
+    "grpc-tools": "^1.12.4",
```

Comment:
Include this library in this repo. It was required in the README as a global install, but including it here makes the README instructions shorter.